cmd/containerd-stress/density.go

```diff
@@ -47,6 +47,14 @@ var densityCommand = cli.Command{
 		},
 	},
 	Action: func(cliContext *cli.Context) error {
+		var (
+			pids  []uint32
+			count = cliContext.Int("count")
+		)
+		if count <= 0 {
+			return nil
+		}
```

Comment:
```suggestion
		if count < 1 {
			return errors.New("count cannot be less than one")
		}
```

Reply:
@samuelkarp 
:smile: , Thank you for dedicating your time to review it.
So , did we still need to notify user , he / she is wrong to fill the parameters..